Purchases/SubscriberAttributes/SpecialSubscriberAttributes.swift

```diff
@@ -7,38 +7,41 @@
 //
 //      https://opensource.org/licenses/MIT
 //
-//  SpecialSubscriberAttributes.swift
+//  ReservedSubscriberAttributes.swift
```

Comment:
Looks like you still need to git rename the `SpecialSubscriberAttributes.swift` to `ReservedSubscriberAttributes.swift`  file 🔠

Reply:
great catch